src/mumble/MainWindow.cpp

```diff
@@ -715,13 +715,35 @@ void MainWindow::saveImageAs() {
 	if (!ok) {
 		// In case fname did not contain a file extension, try saving with an
```

Comment:
Wow. Qt doesn't let us find out whether it actually failed because of this? That kinda sucks...